tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/__bindingIdCamelCase__OSGiTest.java

```diff
@@ -65,6 +69,6 @@ public void tearDown() {
     public void creationOf${bindingIdCamelCase}Handler() {
         assertThat(bridge.getHandler(), is(nullValue()));
         managedThingProvider.add(bridge);
-        waitForAssert(() -> assertThat(bridge.getHandler(), is(notNullValue())));
+        waitForAssert(() -> assertThat(managedThingProvider.get(bridge.getUID()), is(notNullValue())));
```

Comment:
I know, the hamcrest matcher was used before, but while you're here anyway could you please change it to 

```java
waitForAssert(() -> assertNotNull(managedThingProvider.get(bridge.getUID())));
```